src/angular/accordion/expansion-panel.scss

```diff
@@ -83,7 +83,7 @@
   :where(html:not(.sbb-lean)) & {
     border: var(--sbb-border-width-thin) solid var(--sbb-color-storm);
     border-radius: 50%;
-    color: var(--sbb-color-granite);
+    color: var(--sbb-color-black);
```

Comment:
I see that you re-added it, but I think then we should apply it for lean also? Or should it be placed on a parent container?